march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -230,6 +231,12 @@ void MarchHardwareInterface::write(const ros::Time& /* time */, const ros::Durat
   {
     // Enlarge joint_effort_command because ROS control limits the pid values to a certain maximum
     joint_effort_command_[i] = joint_effort_command_[i] * 1000.0;
+    if (abs(joint_effort_command_copy_[i] - joint_effort_command_[i]) > MAX_EFFORT_CHANGE)
```

Comment:
Prefix standard library functions with `std::` so that it is clear they are from the standard library (and not from us)
```suggestion
    if (std::abs(joint_effort_command_copy_[i] - joint_effort_command_[i]) > MAX_EFFORT_CHANGE)
```